phoenix-core/src/main/java/org/apache/phoenix/coprocessor/TaskRegionObserver.java

```diff
@@ -71,6 +71,11 @@
     @GuardedBy("TaskRegionObserver.class")
     // initial delay before the first task is handled
     private static final long  initialDelay = 10000; // 10 secs
+    private static boolean disableTaskHandling = false;
```

Comment:
You still have the `initialDelay` parameter, you could leverage it by exposing out as you did earlier and set it to `Integer.MAX_VALUE`. That approach is followed for the other rebuilder thread task. 
OR
expose a parameter out that disables the complete framework, not just for test purposes. You could use set that parameter to false in tests, defaults to true.

Reply:
Will do it